<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backend</class>
 <widget class="QWidget" name="backend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;后台&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string>客户管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string>菜单管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dealButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string>处理订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string>统计报表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
